main.go

```diff
@@ -116,6 +187,27 @@ func main() {
 	log.Printf("armored-witness-boot: starting kernel@%.8x\n", image.Entry())
 
 	if err = image.Boot(preLaunch); err != nil {
-		panic(fmt.Sprintf("load error, %v\n", err))
+		panic(fmt.Sprintf("armored-witness-boot: load error, %v\n", err))
+	}
+}
+
+func manifestVerifiers() ([]note.Verifier, error) {
+	var manifestKeys []string
+	if err := json.Unmarshal([]byte(OSManifestVerifiers), &manifestKeys); err != nil {
+		return nil, fmt.Errorf("invalid OSManifestVerifiers format: %v", err)
 	}
+	manifestVerifiers := []note.Verifier{}
```

Comment:
Would the size be always equal to the `len(manifestKeys)`?

Reply:
Yup, changed to preallocate the right size.